browser/brave_vpn/vpn_utils.cc

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_vpn/vpn_utils.h"
+
+#include "brave/browser/profiles/profile_util.h"
+#include "brave/components/brave_vpn/brave_vpn_utils.h"
+#include "build/build_config.h"
+
+namespace brave_vpn {
+
+bool IsBraveVPNEnabled(content::BrowserContext* context) {
+  // TODO(simonhong): Can we use this check for android?
+  // For now, vpn is disabled by default on desktop but not sure on
+  // android.
+#if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC)
+  return brave_vpn::IsBraveVPNEnabled() && brave::IsRegularProfile(context);
```

Comment:
if someone enables the VPN in a regular window, it will still apply to private windows, right? this is just making it such that the VPN UI doesn't appear in a private window?

Reply:
@diracdeltas that is correct 😄 The credentials logic and UI won't show/work in a private window (yet). But the actual VPN functionality is fulfilled by the OS and isn't affected (ex: VPN is used for all apps / windows / etc on a system level)